docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -439,7 +439,7 @@ endif::encode_method[]
 ==== Configuration Parameters
 [source,ruby]
 ----------------------------------
-  config :variable_name, :validate => :variable_type, :default => "Default value", :required => boolean, :deprecated => boolean
+  config :variable_name, :validate => :variable_type, :default => "Default value", :required => boolean, :deprecated => boolean, :obsolete => string
```

Comment:
Nevermind.  I see that you're populating `:obsolete` with a value later.

![i-see-what-you-did-there](https://cloud.githubusercontent.com/assets/1020383/10188546/641698b6-6760-11e5-807e-c85ca9ba6d57.jpg)